src/Stratis.Bitcoin.Features.Consensus/StakeValidator.cs

```diff
@@ -362,13 +353,12 @@ private bool VerifySignature(UnspentOutputs coin, Transaction txTo, int txToInN,
         /// </remarks>
         /// <exception cref="ConsensusErrors.StakeTimeViolation">Thrown in case transaction time is lower than it's own UTXO timestamp.</exception>
         /// <exception cref="ConsensusErrors.StakeHashInvalidTarget">Thrown in case PoS hash doesn't meet target protocol.</exception>
-        private void CheckStakeKernelHash(ContextStakeInformation context, ChainedBlock prevChainedBlock, uint headerBits, uint prevBlockTime,
-            BlockStake prevBlockStake, UnspentOutputs stakingCoins, OutPoint prevout, uint transactionTime)
+        private void CheckStakeKernelHash(ContextStakeInformation context, uint headerBits, BlockStake prevBlockStake, UnspentOutputs stakingCoins,
```

Comment:
What all those where not used?
Hmm probably leftover of the V2 deleted code

Reply:
>What all those where not used?

Yeah. 

>Hmm probably leftover of the V2 deleted code

So did @Aprogiena said. He was refactoring this class before.